chapters/02-filesystem.Rmd

```diff
@@ -12,12 +12,9 @@ and filesystem terminology
   - `dir_map()`
   - `file_delete()`, `file_info()`, `file_move()`
 
+## Final exercise {#02-final-exercise}
```

Comment:
For the above do we also want to include cp, mv, rm, and  mkdir?

Reply:
No, we're sticking within the R Console. In a package dev environment in R, you don't really do much outside of R. And for this "intro to RSE" I think the focus should be purely on getting as good as possible in R first.